simple/government_bills/pdftools.py

```diff
@@ -39,7 +40,7 @@ def pdftotext_version():
     p.kill()
     return major, minor, patchlevel
 
-PDFTOTEXT_VERSION = pdftotext_version()
+PDFTOTEXT_VERSION = SimpleLazyObject(lambda: pdftotext_version())
 
 if DEBUG:
```

Comment:
Test fail with:
TypeError: coercing to Unicode: need string or buffer, SimpleLazyObject found